reporting/bootstrap.js

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global APP_SHUTDOWN:false */
+
+"use strict";
+
+const {classes: Cc, interfaces: Ci, utils: Cu} = Components;
+Cu.import("resource://gre/modules/Services.jsm");
+Cu.import("resource://gre/modules/XPCOMUtils.jsm");
+Cu.import("resource://gre/modules/Timer.jsm");
+Cu.import("resource://gre/modules/Log.jsm");
+XPCOMUtils.defineLazyModuleGetter(this, "TelemetryController",
+  "resource://gre/modules/TelemetryController.jsm");
+XPCOMUtils.defineLazyModuleGetter(this, "TelemetryEnvironment",
+  "resource://gre/modules/TelemetryEnvironment.jsm");
+
+const kTelemetryPingDelay = 15000;
```

Comment:
I know neither of us would rather use a timer here, but since we only have one shot at fixing this, it will have to do. However I think a value of 30000 would be better, according to https://mzl.la/2BclhNw. Around 89% of release version sessions get to first paint before 15 msec, but for 30 msec it goes to 93%. Not a huge difference, but perhaps worthwhile?